### DISCUSSION TITLE
trying to play minecraft this error happened

### DISCUSSION THREAD

**Discussion by outofbounds2005** [microsoft/openjdk#631] [Q&A] [answered]
im not really sure if this is the right place but i dont really know what to do to fix it, ive tried uninstalling and reinstalling my graphics drivers and reinstalling curseforge but heres the error
15:34:58.552
game
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007fff243c30dd, pid=4268, tid=20952
15:34:58.552
game
#
15:34:58.552
game
# JRE version: OpenJDK Runtime Environment Microsoft-28056 (17.0.1+12) (build 17.0.1+12-LTS)
15:34:58.552
game
# Java VM: OpenJDK 64-Bit Server VM Microsoft-28056 (17.0.1+12-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
15:34:58.553
game
# Problematic frame:
15:34:58.555
game
# V  [jvm.dll+0x2d30dd]
15:34:58.556
game
#
15:34:58.556
game
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
15:34:58.556
game
#
15:34:58.556
game
# An error report file with more information is saved as:
15:34:58.556
game
# E:\minecraft\Instances\FTB StoneBlock 3\hs_err_pid4268.log
15:34:58.614
game
#
15:34:58.614
game
# If you would like to submit a bug report, please visit:
15:34:58.614
game
#   https://github.com/microsoft/openjdk/issues
[2024-11-29-1.log](https://github.com/user-attachments/files/17963552/2024-11-29-1.log)

**Comment by karianna** [comment, marked as answer]
Thank you for the report!
It looks like you are reporting that the game Minecraft crashed for you. Please try the following:

Update your graphics drivers, see [this Minecraft Help Center article](https://help.minecraft.net/hc/en-us/articles/4409137348877-Minecraft-Java-Edition-Game-Crashes-and-Performance-Issues-FAQ), section "Update your Video Card Drivers";
often Minecraft crashes are caused by outdated graphics drivers
In case you are using mods, try disabling them one by one until you find the mod causing the crash, then report the crash to the mod author
If you are using a third-party launcher, try the [official launcher](https://www.minecraft.net/download) instead
If none of the above helped, and you are still experiencing the crash with the official launcher and without any mods installed, please submit this issue over at the [Mojang bug tracker](https://bugs.mojang.com/projects/MC/summary). Please search for existing reports first; in case you do not find any, create a new report and let us know about the issue number here (e.g. MC-123456).
The Mojang team will take the first look. If an OpenJDK bug is identified, the Mojang team will contact the Microsoft Build of OpenJDK team to address the issue.
